api/config/project/matrixBlockTypes/externalLink--9a5411c3-bbd7-4f75-a62a-498711e141e4.yaml

```diff
@@ -35,11 +35,11 @@ fields:
     searchable: false
     settings:
       maxLength: '255'
-      placeholder: null
+      placeholder: ''
       types:
         - url
     translationKeyFormat: null
-    translationMethod: site
```

Comment:
Was this an intentional change? Did you test the client build with the "translationMethod" changed to "none"?